src/propertyvalueestimate.py

```diff
@@ -5,13 +5,11 @@
 from selenium.webdriver.support import expected_conditions as EC
 from bs4 import BeautifulSoup
 import domainapi as api
+from domainapi import DOMAIN_CLIENT_URL
 
-advertId = 2017019671
-DOMAIN_CLIENT_URL = "https://www.domain.com.au/"
```

Comment:
Moved this to the central domain api file to keep the urls together, note how its imported on the right line 8